libs/features/mas/src/merch-card-collection.js

```diff
@@ -366,8 +366,7 @@ export class MerchCardCollection extends LitElement {
                 hierarchy: [],
                 placeholders: fragment.placeholders,
                 sidenavSettings: sidenavSettings
-            };
-
+            };          
```

Comment:
nit: useless space at the end of bracket
<img width="207" height="58" alt="image" src="https://github.com/user-attachments/assets/94b29c92-0705-4cb9-8c5e-a2e0d6068a4c" />